frontend/src/components/UserSignUp/EventCard.jsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
```

Comment:
FYI it looks like you may have branched off of your EventCard branch instead of main since your changes from the event card in here. Not a super big deal, but it does usually lead to merge conflicts so its something to be aware of.